twenty-first/src/util_types/blake3_wrapper.rs

```diff
@@ -26,6 +28,17 @@ impl From<u128> for Blake3Hash {
     }
 }
 
+/// Automatically dereference our wapper type into the underlying hash.
+/// This allows us to pass a `&blake3_wrapper::Blake3Hash` everywhere we need a `&blake3::Hash`.
+/// [Deref]: https://doc.rust-lang.org/book/ch15-02-deref.html#implicit-deref-coercions-with-functions-and-methods
+impl Deref for Blake3Hash {
+    type Target = blake3::Hash;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
```

Comment:
> *... everywhere we need a `&blake3::Hash`.*

What places are those?

I believe the main purpose of blake3 at this point is for tests that use a lot of hashing to go faster.